main.sh

```diff
@@ -144,6 +177,11 @@ if [ "${arg_d}" = "1" ]; then
   LOG_LEVEL="7"
 fi
 
+# verbose mode
+if [ "${arg_v}" = "1" ]; then
+  set -o verbose
+fi
+
```

Comment:
This hunk is just related to the additional `-v` flag.... could easily remove this and the extra definition in the usage string